src/client/caching/structures/guilds/Guild.ts

```diff
@@ -324,22 +346,121 @@ export class Guild extends Structure {
 		return this.joinedTimestamp === null ? null : new Date(this.joinedTimestamp);
 	}
 
+	/**
+	 * Returns a PNG image URL representing the image widget of the guild.
+	 * @since 0.0.1
+	 * @param options The options for the widget image.
+	 * @see https://discord.com/developers/docs/resources/guild#get-guild-widget-image
+	 */
+	public getWidgetImageURL(options?: WidgetImageOptions): string {
+		const path = Routes.guildWidgetImage(this.id);
+		// TODO(VladFrangu): I think we should move this to @klasa/rest
+		const url = new URL(`https://discord.com/api${path}`);
```

Comment:
Shouldn't this be the CDN.

Reply:
Apparently not: https://discord.com/api/guilds/81384788765712384/widget.png?style=shield